doc/splitting-audio.md

```diff
@@ -29,63 +28,78 @@ Terminal=false
 Type=Application
 ```
 
-- We will be using this application later on to setup output routing.
-- **NOTE**: If you are using a window manager though, make sure to add `qpwgraph <path_to_config> -axm` to your autostart script for your window manager by replacing `<path_to_config>` with the path of the config you save later on during the setup.
-- Eg: In i3, Say a config is in `~/Speedrunning.qpwgraph`, then we put `exec qpwgraph ~/Speedrunning.qpwgraph -axm` in `~/.config/i3/config`.
+We will be using this application later on to setup output routing.  
 
-# Setting up Virtual Audio Cables
+**NOTE**: If you are using a window manager, make sure to add `qpwgraph <path_to_config> -axm` to your autostart script for your window manager by replacing `<path_to_config>` with the path of the config you save later on during the setup.  
+Eg: In i3, say a config is in `~/Speedrunning.qpwgraph`, then we put `exec qpwgraph ~/Speedrunning.qpwgraph -axm` in `~/.config/i3/config`.
 
-- We setup virtual audio cables by using a script that launches each time you login to your system.
-- Copy the following contents into a file named `virtual_inputs` and in a terminal execute `chmod +x virtual_inputs` to make it executable and then move the file into `/usr/bin`.
+# Setting up Virtual Audio Devices
 
-```bash
-#!/bin/sh
-pactl load-module module-null-sink media.class=Audio/Sink sink_name=Virtual_Input_1
-pactl load-module module-null-sink media.class=Audio/Sink sink_name=Virtual_Input_2
+`virtual-input-1` and `virtual-input-2` can be renamed to make things more easily identifiable for yourself. But for the purpose of this guide, I'll be using those two names.
+
+We're going to use `.conf` files to configure virtual devices. These files should be placed in `~/.config/pipewire/pipewire.conf.d` (for user configuration) or `/etc/pipewire/pipewire.conf.d` (for system-wide configuration).
+
+First, add a file `virtual-input-1.conf` with the following contents.
+```
+context.objects = [
+	{	factory = adapter
+		args = {
+			factory.name = support.null-audio-sink
+			node.name = "virtual-intput-1.conf"
+			media.class = Audio/Sink
+		}
+	}
+]
 ```
 
-- If you don't have the `pactl` command you can install it by executing `sudo dnf install pulseaudio-utils`.
-- The above script is to make 2 virtual inputs which are more than enough to do basic audio splitting. You can add more by just copying and pasting these lines and changing the `sink_name` to `Virtual_Input_3` and so on.
-- Now we can setup `virtual_inputs` to launch on startup.
-- Copy the following contents into a file named `virtual_inputs.desktop` and drop it in `~/.config/autostart/`.
+Next, add another file `virtual-input-2.conf` with the following contents.
 
 ```
-[Desktop Entry]
-Encoding=UTF-8
-Type=Application
-Exec=virtual_inputs
-Terminal=false
-Name=Virtual Inputs
+context.objects = [
+	{	factory = adapter
+		args = {
+			factory.name = support.null-audio-sink
+			node.name = "virtual-input-2.conf"
```

Comment:
I'd prefer the names be `virtual-input-n` only tbh

Reply:
I forgot I accidentally added a `.conf` at the end of the names my bad